controller/utils.go

```diff
@@ -845,3 +889,366 @@ func validateMetricsTrigger(configYAML *models.ConfigYAML, logger logrus.FieldLo
 	}
 	return nil
 }
+
+func schedulerAndConfigFromName(
```

Comment:
this and all functions below it are really new functions?

Reply:
They already existed in controller. I just separated the code in functions and put it in utils

Reply:
got it! 💯